src/crypto.js

```diff
@@ -148,6 +149,34 @@ class Crypto {
     }
   }
 
+  async feesRate() {
+    const cryptos = ['bitcoin', 'bitcoin-cash', 'dogecoin', 'dash', 'litecoin'];
+    const res = [];
+    res.push('These are fee rates for some cryptocurrencies:');
+    for (const crypto of cryptos) {
+      res.push(crypto.toUpperCase());
+      const options = {
```

Comment:
You can create object outside of for of cycle and just update path field. Also tryo using for await